### PR TITLE
Add support for the new support file on github by copy the support section of the readme

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Add support for the new support file on github by copy the support section of the readme.

https://github.com/blog/2400-support-file-support

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on b67dd502b45f161316d9a7c39f1f9d060f88f87e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17538">issues.joomla.org/tracker/joomla-cms/17538</a>.</sub>

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Where can you get support and help?
+====================
+* [The Joomla! Documentation](https://docs.joomla.org/Special:MyLanguage/Main_Page);
+* [Frequently Asked Questions](https://docs.joomla.org/Special:MyLanguage/Category:FAQ) (FAQ);
+* Find the [information you need](https://docs.joomla.org/Special:MyLanguage/Start_here);
+* Find [help and other users](https://www.joomla.org/about-joomla/create-and-share.html);
+* Post questions at [our forums](https://forum.joomla.org);
+* [Joomla Resources Directory](https://resources.joomla.org/) (JRD).
```
Comment (Quy) [inline .github/SUPPORT.md]: Line 8 has an ending `/` after the URL but not on line 7. 
Is it necessary to add a semicolon after each entry?

Comment (brianteeman) [reply .github/SUPPORT.md]: The / doesn't matter

Comment (Quy) [reply .github/SUPPORT.md]: I understand. For consistency sake, either have it in both places or not.

Comment (zero-24) [reply .github/SUPPORT.md]: pushed thanks

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on da998705cbd5fe44455e392487af568dfa639515<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17538">issues.joomla.org/tracker/joomla-cms/17538</a>.</sub>

**Comment by ghost** [comment]
@brianteeman can you please retest?

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on da998705cbd5fe44455e392487af568dfa639515<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17538">issues.joomla.org/tracker/joomla-cms/17538</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Pull request merged**